packages/block-editor/src/store/selectors.js

```diff
@@ -3068,6 +3068,12 @@ export const getBlockEditingMode = createRegistrySelector(
 					return 'contentOnly';
 				}
 
+				// All controlling blocks are treated as content only
+				// by default.
+				if ( areInnerBlocksControlled( state, clientId ) ) {
+					return 'contentOnly';
+				}
+
```

Comment:
This is in zoomed out? I think this needs to be considered, as zoomed out currently has no content editing, and the controlled block could be buried in a section as a piece of content.

Reply:
Should we have a new `role` to declare support for this "controlled content only"? `role: controlled`? Or is detecting controlled inner blocks reliable enough?

I think this specific conditional for Zoom Out can be easily added removed based on the required UX.